src/components/Map.tsx

```diff
@@ -299,43 +262,45 @@ const Map = ({
           animationDuration={animationDuration}
         />
         <MapboxGL.Images images={mapImages} />
-        <MapboxGL.ShapeSource
-          id="networkHotspots"
-          shape={shapeSources.networkHotspotFeatures}
-          onPress={onShapeSourcePress}
-        >
-          <MapboxGL.CircleLayer
-            id="markerNetwork"
-            style={styles.markerNetwork}
-          />
-        </MapboxGL.ShapeSource>
-        <MapboxGL.ShapeSource
-          id="ownedHotspots"
-          shape={shapeSources.ownedHotspotFeatures}
-          onPress={onShapeSourcePress}
-        >
-          <MapboxGL.CircleLayer id="markerOwned" style={styles.markerOwned} />
-        </MapboxGL.ShapeSource>
-        <MapboxGL.ShapeSource
+        <H3Grid bounds={mapBounds} visible={showH3Grid} />
+        <HotspotsCoverage
```

Comment:
There seems to be some layering issues here. I wonder if we could somehow simplify by using the `<Coverage/>` component.

Reply:
I think they are all resolved, I would prefer to leave it as is because I like the modular approach of multiple layers so that the coverage component doesn't get inflated with everything from the network map. It will also help with customization down the road is discovery needs changes the network map doesn't. If we find problems I'm open to changing it, but I think its working well.